### PR TITLE
Update policies to reflect new 3 level structure

### PR DESCRIPTION
**Pull request by moederballa**

### CODE REVIEW

File: policies/aws_cis_s3_policy.json [hunk 1]
```diff
@@ -108,8 +121,9 @@
           "checks": {
             "logging enabled": [
               {
-                "check": "regex",
-                "expected": ".",
+                "exp": "false",
+                "check": "regex_excludes",
+                "expected": "false",
```
Comment (moederballa) [inline policies/aws_cis_s3_policy.json:126]: The agent handles this now but setting the value to false if it's empty or invalid.

File: policies/aws_cis_ebs_policy.json [hunk 2]
```diff
@@ -30,10 +30,16 @@
...
           "ci_path": [
+            "Encryption by Default",
             "Encryption by Default",
             "Encryption"
           ],
           "check_type": "encryption by default",
+          "selectList": [
+            "{\"EBS encryption by default\":false}",
+            "{\"EBS encryption by default\":false}",
+            "*"
```
Comment (moederballa) [inline policies/aws_cis_ebs_policy.json:41]: Not completely sure what all these `selectList` things are and why they're here ???

Comment (mccarths) [reply policies/aws_cis_ebs_policy.json:41]: they can all be removed, it's only used by the UI. They shouldn't be part of the export.

File: policies/aws_cis_vpc_policy.json [hunk 3]
```diff
@@ -1,51 +1 @@
...
...rses the VPC and can be used to detect anomalous traffic or insight during security workflows.","remediation":"Perform the following to determine if VPC Flow logs is enabled:\nFrom Console:\n1. Sign into the management console\n2. Select Services then VPC\n3. In the left navigation pane, select Your VPCs\n4. Select a VPC\n5. In the right pane, select the Flow Logs tab.\n6. If no Flow Log exists, click Create Flow Log\n7. For Filter, select Reject\n8. Enter in a Role and Destination Log Group\n9. Click Create Log Flow\n10. Click on CloudWatch Logs Group\nNote: Setting the filter to \"Reject\" will dramatically reduce the logging data accumulation for this recommendation and provide sufficient information for the purposes of breach detection, research and remediation. However, during periods of least privilege security group engineering, setting this the filter to \"All\" can be very helpful in discovering existing traffic flows required for proper operation of an already running environment."}]},"ci_path":["Flow Logs","Flow Logs","*"],"check_type":"other","path_error":"","description":"3.9 Ensure VPC flow logging is enabled in all VPCs","nodeGroupsOpen":true}]}]}],"scan_options":{}}
```
Comment (mccarths) [inline policies/aws_cis_vpc_policy.json:1]: Pretty format please

**Review by mccarths — APPROVED** [review]

**Pull request merged**